README.md

```diff
@@ -19,7 +19,9 @@
 * __[DOM components and props](#dom-components-and-props)__
 * __[Building custom components](#building-custom-components)__
     * __[Component Boilerplates](#component-boilerplate-templates)__
+    * __[Component Formatting](#component-formatting)__
```

Comment:
See my penultimate comment; may be good to have a "component formatting" set of guidelines in the first level of this list, and then "component best practices" under this "Building custom components" section.